packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -140,14 +140,14 @@ export default class QuanticFacet extends LightningElement {
 
   get facetSearchResults() {
     const results = this.facet.state.facetSearch.values;
-    return results.map((result) => ({
+    return results?.map((result) => ({
```

Comment:
If I am not mistaken, this either returns results or `undefined` right?

If a getter that is used directly in an iterator/for:each in your templates ever return something other than an array, the whole LWC will throw and not load.

I would say, we need to make sure these functions `get facetSearchResults()` either return results or an empty array.